src/main/scala/util/DescribedSRAM.scala

```diff
@@ -29,14 +29,17 @@ object DescribedSRAM {
       case d => d.getWidth
     }
 
+    val f = () => mem.toNamed.toString.hashCode
```

Comment:
The odds of a hash collision are low since we're likely to have 10s of these, but it's worth thinking about. It might be worth using a 64-bit hash

Reply:
as discussed through Slack, the ID generator has been switched to a global counter